examples/workdir/inventory/hosts_combined_example.yml

```diff
@@ -0,0 +1,76 @@
+---
+caasp-admin:
+  hosts:
+    ccpci-27-admin-ekulfxaq55em:
+      ansible_host: 10.86.3.236
+  vars:
+    ansible_user: root
+caasp-masters:
+  hosts:
+    ccpci-27-master-0:
+      ansible_host: 10.86.3.87
+  vars:
+    ansible_user: root
+caasp-workers:
+  hosts:
+    ccpci-27-worker-0:
+      ansible_host: 10.86.3.206
+    ccpci-27-worker-1:
+      ansible_host: 10.86.3.187
+    ccpci-27-worker-2:
+      ansible_host: 10.86.3.220
+  vars:
+    ansible_user: root
+soc-deployer:
+  hosts:
+    soc-aio:
+      ansible_host: 10.86.3.69
+  vars:
+    ansible_user: root
+ses_nodes:
```

Comment:
ses_nodes has caused errors in airship deployments. Has this been resolved ?

Reply:
What kind of errors, are they format related i.e  specifying one node vs specifying multiple nodes?. I do have ses_nodes (single node) in my manual deployment, which seemed to work fine. From airship CI point of view, it will use a single node ses deployment too.  I haven't yet succeeded in getting all the way through a complete aiship deployment, still in deploy fix cycle.

Reply:
Last time I ran run.sh deploy_airship with ses_nodes in my inventory it tried to update docker on ses_node and failed.  If it is still a problem, it is not something for this review. Just wanted to bring it up.